src/messages/javascript.js

```diff
@@ -126,6 +126,13 @@ export const UNSUPPORTED_API = {
   description: i18n._('This API has not been implemented by Firefox.'),
 };
 
+export const REMOVED_MV2_API = {
+  code: 'UNSUPPORTED_API',
+  message: null,
+  messageFormat: i18n._('"{{api}}" has been removed in manifest_version 3'),
```

Comment:
nit: if this is a user message, maybe we should write "Manifest Version" instead of `manifest_version`

Reply:
@willdurand if for user message we mean that this message is meant for the extension developer, the answer is yes. I personally preferred `manifest_version` here because it was matching the name of the property in the manifest that the error want to point the attention of the developer to.

I'm not sure that 'Manifest Version' would always be read as we expect, e.g. some developer may read it as the "version" property in the Manifest (which is definitely not what this error means).

Reply:
I think it's valid, maybe we should have both then?

```suggestion
  messageFormat: i18n._('"{{api}}" has been removed in Manifest Version 3 (`manifest_version` property)'),
```

Reply:
:+1: sgtm